src/ex1/script.js

```diff
@@ -0,0 +1,64 @@
+const listToDo = document.querySelector(".listToDo");
+const count = document.querySelector(".count");
+const buttonClearAll = document.querySelector(".buttonClearAll");
+const newToInput = document.querySelector(".newToInput");
+const buttonAdd = document.querySelector(".buttonAdd");
+
+var removeSVG = '<i class="fa fa-trash"></i>';
+count.textContent = 0;
+
+clickButtonClearAll();
+
+document.addEventListener("keydown", addEnter);
+buttonAdd.addEventListener("click", add);
+buttonClearAll.addEventListener("click", deleteListToDo);
+
+function addEnter(event) {
+  if (
+    event.code === "Enter" &&
+    document.querySelector("#newToDo input").value.length != 0
+  )
+    add();
+}
+
+function deleteListToDo() {
+  if (confirm("Are you sure??") == true) {
+    document.querySelector("#listToDo").innerHTML = "";
+    count.textContent = 0;
+    clickButtonClearAll();
+  }
+}
+
+function clickButtonClearAll() {
+  if (count.textContent > 0) {
+    buttonClearAll.classList.add("active");
+  } else {
+    buttonClearAll.classList.remove("active");
+  }
+}
+
+function add() {
+  if (document.querySelector("#newToDo input").value.length == 0) {
+    alert("Please Enter new todo");
+  } else {
+    var div = document.createElement("div");
+    div.classList.add("todo");
+    var item = document.createElement("span");
+    item.innerText = document.querySelector("#newToDo input").value;
+    var remove = document.createElement("button");
+    remove.classList.add("delete");
+    remove.innerHTML = removeSVG;
+    remove.addEventListener("click", function () {
+      this.parentNode.style.opacity = "0";
+      count.textContent--;
+      clickButtonClearAll();
+      setTimeout(() => this.parentNode.remove()(), 200);
```

Comment:
is that for the animation? cool idea but it introduce a bug, if I double click remove button the counter is getting out of sync and you can get to this situation
![image](https://user-images.githubusercontent.com/65164924/169856291-01bd7abc-681c-4af4-99da-5309eecc5bb1.png)